### DISCUSSION TITLE
[Announcement] Threat hunt award session

### DISCUSSION THREAD

**Discussion by KihongHeo** [prosyslab-classroom/cs348-information-security#467] [Announcements]
Hi all,

Every Monday, we will have an award session for threat hunt winners of the week (if exists).

Tanapoom, Suchan, Seoyeon, thanks for your submission and hope to see you then.
See the results [here](https://github.com/prosyslab-classroom/cs348-information-security/blob/main/hof.md#2023-spring).

Other submissions (after Mar 15) will be graded next week.

**Comment by tncks0121** [comment]
Hello, I'm sorry for not being in time due to unexpected traffic. It seems the vulnerabilities are disclosed in the photo album while they are classified in `hof.md`, is that intended?

**Comment by KihongHeo** [comment]
No problem.
